naga/src/back/spv/index.rs

```diff
@@ -11,16 +11,31 @@ use crate::{arena::Handle, proc::BoundsCheckPolicy};
 
 /// The results of performing a bounds check.
 ///
-/// On success, `write_bounds_check` returns a value of this type.
+/// On success, `write_bounds_check` returns a value of this type. The
```

Comment:
nitpick: RE: the original code: it'd be nice if this used a link (``[`write_bounds_check`]``) for both Rustdoc browsing and avoiding potential rename rot.

Reply:
Pushed myself as 390ccb6a.